v2/internal/pkg/release/cincinnati.go

```diff
@@ -123,29 +125,45 @@ func (o *CincinnatiSchema) GetReleaseReferenceImages(ctx context.Context) ([]v2a
 	// before making a deep copy
 	// check that the "platform.release" field is not empty
 	if len(o.Config.Mirror.Platform.Release) > 0 {
-		copyImage := v2alpha1.CopyImageSchema{
-			Source:      o.Config.Mirror.Platform.Release,
-			Destination: "",
-			Origin:      o.Config.Mirror.Platform.Release,
-		}
-		allImages = append(allImages, copyImage)
 		// OCPBUGS-50617
-		// include signature verify and download for rc (release candidate) by digest
-		// skip ec (engineering candidate) and other releases (by tag)
+		// include signature verify and download for ga releases, rc (release candidate)
+		// and ec Iengineering candidate) by tag or digest
 		imgSpec, err := image.ParseRef(o.Config.Mirror.Platform.Release)
 		if err != nil {
 			return []v2alpha1.CopyImageSchema{}, err
 		}
 		if imgSpec.IsImageByDigestOnly() {
+			copyImage := v2alpha1.CopyImageSchema{
+				Source:      o.Config.Mirror.Platform.Release,
+				Destination: "",
+				Origin:      o.Config.Mirror.Platform.Release,
+			}
+			allImages = append(allImages, copyImage)
 			imgs, err := o.Signature.GenerateReleaseSignatures(ctx, allImages)
 			if err != nil {
 				return []v2alpha1.CopyImageSchema{}, err
 			}
 			return imgs, nil
 		} else {
-			o.Log.Warn("release image is not by digest: SKIPPING signature verification")
+			srcImage := o.Opts.Global.NewSystemContext()
+			digest, err := o.Manifest.GetDigest(context.Background(), srcImage, imgSpec.ReferenceWithTransport)
```

Comment:
```suggestion
			digest, err := o.Manifest.GetDigest(ctx, srcImage, imgSpec.ReferenceWithTransport)
```
Any reason for creating a new `Background` context?

Reply:
Ah yes - good catch :)